### DISCUSSION TITLE
Display Country Only

### DISCUSSION THREAD

**Discussion by kewlfrogz** [Daniel-Ioannou/flutter_country_picker#23] [Q&A] [answered]
Thank you for creating a very easy to implement Country Picker for Flutter.
I made showPhoneCode: false, and it does not display the phone code on the drop down list; however, it posts that data back to the context; United States (US) [+1], and I would rather have it show just show the Country when posting back after selection. Would it be possible to select only the country name from country.displayname?

Thanks again.

**Comment by Daniel-Ioannou** [comment, marked as answer]
Country Model have all the info you need for the selected country.

Here is some useful:
- `country.name` :  The country name in English || e.g: United Kingdom
- `country.countryCode` :  The country code, ISO (alpha-2) || e.g: GB
- `country.phoneCode` :  The country phone code || e.g: 44
- `country.displayname` :  country name (country code) [phone code] || e.g: United Kingdom (GB) [+44]
- `country.displayNameNoE164Cc` :  country name (country code) || e.g: United Kingdom (GB)

If you want only the country name use `country.name` or if you want the country name and country code use `country.displayNameNoE164Cc`

@kewlfrogz Thanks for the question. If you have any additional questions or suggestions please feel free to ask.
